packages/app/script-utils/src/vault/syncEnvWithVault.ts

```diff
@@ -13,7 +13,7 @@ import { vaultGetVars, vaultLogin } from './vault.ts'
  * @param envContents String contents of the .env file
  * @param parsedEnv Record of .env file key-values (parsed using `dotenv.parse`)
  */
-const upsertEnvValue = (
+export const upsertEnvValue = (
```

Comment:
This function is currently just copied and pasted into the autopilot setup scripts word-for-word so I thought better to expose it and have a single source of truth.